### PR TITLE
feat: Feature/front page

### PR DESCRIPTION
**Pull request by loldemort**

### CODE REVIEW

**Comment by psukys** [comment]
WIP?

**Comment by loldemort** [comment]
@psukys work in progress

File: src/app/components/frontpage/frontpage.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/**
```
Comment (shumail92) [inline src/app/components/frontpage/frontpage.js]: Please remove this file. We don't create a .js file in components. The component .ts file is where everythign should go on.

**Comment by shumail92** [comment]
Also please try to squash commits if possible.

**Pull request merged**